generate-index.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+import json
+import os
+
+
+def generate_index(root):
+    os.chdir(root)
+    index = []
+    for model in sorted(os.listdir(".")):
+        if not os.path.isdir(model):
+            continue
+        os.chdir(model)
+        model_contents = os.listdir(".")
+        gltf_variant_dirs = [d for d in model_contents if d.startswith("glTF")]
+        variants = {}
+        for variant_dir in gltf_variant_dirs:
+            model_file = [f for f in os.listdir(variant_dir)
+                          if f.endswith(".glb") or f.endswith(".gltf")][0]
+            variants[variant_dir] = model_file
+        if "screenshot" not in model_contents:
+            print "WARNING: no screenshot found for {}".format("model")
```

Comment:
So I just merged this, and then I found a problem.  On line 21 here, `model` should not be in quotes.  Also, Python 3.x requires the `print` arguments to be in parentheses, as it changed from a statement to a function.  Can you open a new PR for that?  Thanks!

Reply:
Ok, will do.

Reply:
Addressed in #168.